core/src/main/scala/org/genivi/sota/core/Boot.scala

```diff
@@ -83,6 +82,10 @@ object Boot extends App with DatabaseConfig {
     case _ => {
       val notifier = DefaultUpdateNotifier
       Http().bindAndHandle(routes(notifier), host, port)
```

Comment:
Yes, start webservice api with vehicles route as it is right now. Add new empty route for new Vehicle-Update API.